### PR TITLE
Update data-lake-analytics.md

### PR DESCRIPTION
**Pull request by JeremyLikness**
Updated per requests from Cam and Shayne. @CamSoper  @spboyer - added general link to samples and fixed types in examples

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/Azure/azure-docs-sdk-dotnet, working directory: W:\jgqt-s, target branch: master, target branch commit: 6ecd91094ff6e15f1829215f9d05d7631dc0ea6c, source branch: 30ade1ee12786f12b14d17f20c427a17/patch-1, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707182000526605-236%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/data-lake-analytics.md [hunk 1]
```diff
@@ -37,47 +37,37 @@ dotnet add package Microsoft.Azure.Management.DataLake.Analytics
...
+/// Setup authentication for this demo
+Authentication auth = new Authentication("microsoft.onmicrosoft.com"); // change this to YOUR tenant
+auth.Authenticate();
 
-```bash
-dotnet add package Microsoft.Azure.Management.DataLake.Analytics
-```
-
-### Example
+// Collect info about the Azure resources needed for this demo
```
Comment (spboyer) [inline api/overview/azure/data-lake-analytics.md]: remove this section

File: api/overview/azure/data-lake-analytics.md [hunk 2]
```diff
@@ -37,47 +37,37 @@ dotnet add package Microsoft.Azure.Management.DataLake.Analytics
...
+string subid = "<Subscription Id>";
+string rg = "<Resource Group>";
+string adla_name = "<user Name>";
 
---Example overview--
+// Identify the accounts
+AnalyticsAccountRef adla_account = new AnalyticsAccountRef(subid, rg, adla_name);
```
Comment (spboyer) [inline api/overview/azure/data-lake-analytics.md]: Change to: 

`AnalyticsAccountRef adla_account = new AnalyticsAccountRef(subscriptionId, resourceGroup, userName);`

**Comment by JeremyLikness** [comment]
@spboyer I made the requested updates if you want to do final pass review.

**Comment by spboyer** [comment]
#sign-off

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-1/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-1/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\0xjx-o-110128dd\raw-\kfdsbahn.htt\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\0xjx-o-110128dd\raw-\kfdsbahn.htt\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C19%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707191602357986-236%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**